app/scripts/views/mixins/signin-mixin.js

```diff
@@ -20,19 +20,21 @@ define(function (require, exports, module) {
       'click': '_engageSignInForm',
       'input input': '_engageSignInForm'
     },
+
     /**
      * Sign in a user
      *
-     * @param {Account} account
+     * @param {Account} account - account being signed in to
      *     @param {String} account.sessionToken
      *     Session token from the account
      * @param {string} [password] - the user's password. Can be null if
      *  user is signing in with a sessionToken.
+     * @param {string} [unblockCode] - an unblock code.
      * @return {object} promise
      */
-    signIn: function (account, password) {
+    signIn (account, password, unblockCode) {
```

Comment:
not sure how i feel about `unblockCode` being 3rd arg. should this be 
` signIn (account, password, options) {` then `options.unblockCode`?